src/helpers/validators/profile.js

```diff
@@ -0,0 +1,45 @@
+import Joi from '@hapi/joi';
+
+const min = 3;
```

Comment:
there is a middleware to validate the body emptiness, why didn't u use it instead of creating your own?